### PR TITLE
Updates to account for repo name change

### PR DESCRIPTION
**Pull request by bmbouter**
This updates two main things:
- various docs
- the encrypted key for Travis to publish to PyPI. This is an asset that
  is encrypted per-repo on Travis so changing the repo requires
  regenerating it.

https://pulp.plan.io/issues/4444
re #4444

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/en/3.0/nightly/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
         distributions: sdist bdist_wheel
         user: pulp
         password:
-          secure: "ej2lekfOr8S3Du6D6HPlB20+g5jwgZz75ouHlqIpc+IAgdAxmpn5vkspsAevMtchLw7sNgDAAbl0bQVacMDnn+zbo21s93OXcOmYzgy4IlTgopc+xi1XaKFdLMLiEiZ+KOGVJgg7yQwS3yeS28FARZGNDYTF/+N7eITteDThcG4="
+          secure: "EerGV8Z0jybfuq2yHWrD2fu6NnZxLBjAz/kyhGGuXOcSCkBsWfrDoatszMhKtBR7A4o+xzUo/OsRV2rILz3IWPg0aOBk3GAAH0dg6axt25JrJbnd8+lCj4XTyhg0dR/J/EsE0S8xGV6mG6Z+ubRvBRtc1pGdQvq2PxuQcPISi35zNhwqUoNIJ8f1NDa7Rl/7jyhM/YIlrfH2gtRaIDIhrHHeXqm9JIHCPRpyx//t1BKDov3TrPeiohmu/OvwNS3F4a/Juxscl9sAm4JeduJA5N6nYkjlCsYYe7L54kluXIN0bCcKv0qKNDbf/EfTP3/SHzywkJwr5IFlRsf2rNiOiBrec03Xhh10JbIhyFMyI2BqDHSghfxG8D3m4EFtwebowgXixCBBRDyIRxiK+NEt4DxVahwV4gZiYZzG0h7TqL2/zmgTL4Tj09iqKbfLtzccmVLsGikHWCQxctefHCyD6oN/TWRSay0fLZGIyCL0YwQOTyS6IzfBYlwxDtke92okHrH2bASVBczAZcOMUOcbxxYVrL2XWQI8XIjguOBlxbv80GM1ZXC+rEvZ9WOSLBv9QJAC1V+cYfWAjCQ3jpmzz2TSIx4HH7l1wt5fBOMnnWDB+gKfqLlOpMrAnqkbR/YiyXkL0wBtAOW7ZTS0iDARgJpKT3V/dvyqXYwEmWfDJB4="
```
Comment (asmacdo) [inline .travis.yml]: Any reason for this change?

Comment (bmbouter) [reply .travis.yml]: @asmacdo you're gonna love this. See the commit message :)

File: docs/release-notes/pulpcore/3.0.x.rst [hunk 2]
```diff
@@ -53,43 +53,43 @@ Breaking Changes
...
 ========
 
-`Comprehensive list of changes and bugfixes for beta 11 <https://github.com/pulp/pulp/compare/pulpcore-3.0.0b10...pulpcore-3.0.0b11>`_.
+`Comprehensive list of changes and bugfixes for beta 11 <https://github.com/pulp/pulpcore/compare/pulpcore-3.0.0b10...pulpcore-3.0.0b11>`_.
 
 3.0.0b10
 ========
 
-`Comprehensive list of changes and bugfixes for beta 10 <https://github.com/pulp/pulp/compare/pulpcore-3.0.0b9...pulpcore-3.0.0b10>`_.
+`Comprehensive list of changes and bugfixes for beta 10 <https://github.com/pulp/pulpcore/compare/pulpcore-3.0.0b9...pulpcore-3.0.0b10>`_.
```
Comment (asmacdo) [inline docs/release-notes/pulpcore/3.0.x.rst]: This link shows no diff?

Comment (asmacdo) [reply docs/release-notes/pulpcore/3.0.x.rst]: Without testing all the links im not sure, but it looks like the newest ones work, but the older ones don't have a diff.

Comment (bmbouter) [reply docs/release-notes/pulpcore/3.0.x.rst]: Nice find! It's because we changed the tag names and we need to update them. I'm fixing and I'l push.

**Pull request merged**